examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+       "3     2022-05-17 04:00:30+00:00 2022-05-17 06:00:30+02:00   \n",
+       "4     2022-05-17 04:00:40+00:00 2022-05-17 06:00:40+02:00   \n",
+       "...                         ...                       ...   \n",
+       "99699 2022-05-28 16:59:10+00:00 2022-05-28 18:59:10+02:00   \n",
+       "99700 2022-05-28 16:59:20+00:00 2022-05-28 18:59:20+02:00   \n",
+       "99701 2022-05-28 16:59:30+00:00 2022-05-28 18:59:30+02:00   \n",
+       "99702 2022-05-28 16:59:40+00:00 2022-05-28 18:59:40+02:00   \n",
+       "99703 2022-05-28 16:59:50+00:00 2022-05-28 18:59:50+02:00   \n",
+       "\n",
+       "       Extruder A Schmelzedruck  \n",
+       "0                         387.0  \n",
+       "1                         388.0  \n",
+       "2                         388.0  \n",
+       "3                         388.0  \n",
+       "4                         388.0  \n",
+       "...                         ...  \n",
+       "99699                     338.0  \n",
+       "99700                     338.0  \n",
+       "99701                     338.0  \n",
+       "99702                     338.0  \n",
+       "99703                     338.0  \n",
+       "\n",
+       "[99704 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data['time_local'] = data['time'].dt.tz_convert('Europe/Berlin')\n",
+    "data[['time','time_local', 'Extruder A Schmelzedruck']]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0239a449-efc0-4cae-a233-5987c6a45968",
+   "metadata": {},
+   "source": [
+    "### For large periods and more fine-grained control - get_timeseries_with_resampling()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88bd9db6-6561-480f-bc96-ccddbe68cbcd",
+   "metadata": {},
+   "source": [
+    "The `get_timeseries` method returns the time series in the currently highest resolution (10 seconds). For a long period and many variables, this quickly results in a large amount of data. Due to the size of the data, there might be a noticeable delay in receiving the results, and in some cases, it may even cause a request timeout.\n",
+    "\n",
+    "The `get_timeseries_with_resampling` method allows for a higher control over the way in which the data is retrieved. By specifying a resampling method and a resampling interval, the amount of data can be adjusted for the use case. Resampling happens on the server-side and thus reduces the payload.\n",
+    "\n",
+    "\n",
+    "A separate resampling method must be specified for each variable. This is necessary because not every resampling method is available for every data type. For example, categorical data types such as strings can only be aggregated with 'first', 'last' or 'count'. You can find all available resampling methods [in the documentation](https://enlyze-python.readthedocs.io/en/latest/models.html#enlyze.models.ResamplingMethod)."
```

Comment:
```suggestion
    "A separate resampling method must be specified for each variable. This is necessary because not every resampling method is available for every data type. For example, categorical data types such as strings can only be aggregated with `first`, `last` or `count`. You can find all available resampling methods [in the documentation](https://enlyze-python.readthedocs.io/en/latest/models.html#enlyze.models.ResamplingMethod)."
```

Reply:
2d5a669